var/spack/repos/builtin/packages/nwchem/package.py

```diff
@@ -52,7 +56,9 @@ class Nwchem(Package):
     depends_on("lapack")
     depends_on("mpi")
     depends_on("scalapack")
-    depends_on("fftw-api")
+    depends_on("fftw-api", when="+fftw3")
```

Comment:
```suggestion
    depends_on("fftw-api@3", when="+fftw3")
```
?

Reply:
Previous developer did not specify the version of fftw api; I am not sure about if NWChem relies on the version, as NWChem is also aged. Maybe changing the name of the option to "fftw" is better?

Reply:
I think it is indeed FFTW3 which is needed: https://github.com/nwchemgit/nwchem/blob/118d5ffa8b591a2fe725980aab832e1ef1bcd571/src/config/makefile.h#L3484-L3493

Reply:
Thanks for the infomation. I would change the fftw-api version to explicit @3 .